src/styles/Book.css

```diff
@@ -0,0 +1,157 @@
+#book-panel {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  margin: 0 12% 1.5rem 12%;
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+}
+
+.book-box {
+  padding: 1.5rem 0 1.75rem 1.68rem;
+  width: 15rem;
+}
+
+.book-cat {
+  font-size: 0.875rem;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  color: #888;
+}
+
+.book-title {
+  font-size: 1.5rem;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: bold;
+  color: #121212;
+  letter-spacing: -0.2px;
+}
+
+.book-author {
+  font-size: 0.875rem;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: 300;
+  color: #4386bf;
+  margin-top: -0.8rem;
+}
+
+/* BUTTONS */
+
+.buttons-div {
+  margin-top: 1.3rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.book-buttons {
+  font-size: 0.875rem;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: 300;
+  color: #4386bf;
+  border: none;
+  background-color: transparent;
+}
+
+.remove-btn:hover {
+  cursor: pointer;
+  font-weight: 400;
+  color: #1c65a5;
+}
+
+.comment-btn {
+  padding-right: 1.2rem;
+  border-right: 1px solid #e8e8e8;
+}
+
+.edit-btn {
+  padding-left: 1.2rem;
+  border-left: 1px solid #e8e8e8;
+}
+
+/* RIGHT SIDE OF PANEL */
+
+.progress-div {
+  display: flex;
+  padding-right: 10%;
+}
+
+.percentage-box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-right: 5rem;
+}
+
+.status-box {
+  display: flex;
+  flex-direction: column;
+  padding-right: 5rem;
+  border-right: 1px solid #e8e8e8;
+}
+
+.percentage {
+  font-size: 2rem;
+  font-family: 'Montserrat', sans-serif;
+  color: #121212;
+}
+
+.percentage-status {
+  font-size: 0.875rem;
+  font-family: 'Montserrat', sans-serif;
+  color: #888;
+}
+
+.chaper-box {
+  display: flex;
```

Comment:
### [Optional]
I notice you repeated `display: flex` here

We could practice DRY code by doing this;
```
.progress-div, .status-box, .chaper-box, #book-panel {
 display: flex;
}
```